src/CarbonAware.Aggregators/test/CarbonAwareAggregatorTests.cs

```diff
@@ -202,4 +203,77 @@ public void TestGetCurrentForecastDataAsync_InvalidStartEndTimes_ThrowsException
         };
         Assert.ThrowsAsync<ArgumentException>(async () => await Aggregator.GetCurrentForecastDataAsync(props));
     }
+
+    [TestCase("eastus", "2021-11-18T00:00:00Z", "2021-11-18T08:00:00Z", ExpectedResult = 60)]
```

Comment:
Can we name these test cases using `TestName = "my test"` param to easily see what conditions they are exercising?

Reply:
Something we have started doing that you might consider is using this format for TestName values:
"<MethodUnderTest> <returns expected / throws> for: <test case variation description>"

EG "CalculateAverageCarbonIntensityAsync returns expected values for: valid region, data within time period"

Reply:
We could -- the tests were ported with only minimal functional changes/tweaks from _SciScoreAwareAggregetorTest_, and didn't take to "friendly-name" them

Reply:
Ah yes, I forgot this was ported.  Still nice to leave it better than we found it, but that definitely makes this an if-you-have-time-NIT.